test/node_auth.sh

```diff
@@ -3,54 +3,39 @@
 source bin/bacalhau.sh
 
 testcase_node_can_connect_without_token() {
-    subject bacalhau config set node.network.type nats
-    create_node requester
+    create_node orchestrator
 
     create_node compute
     # If this returns successfully, the node started and authenticated.
 }
 
 testcase_preconfigured_token_not_printed() {
-    subject bacalhau config set node.network.type nats
-    subject bacalhau config set node.network.authsecret kerfuffle
+    subject ${BACALHAU} config set orchestrator.auth.token kerfuffle
     assert_equal 0 $status
 
-    create_node requester
-    assert_equal 0 $status
-
-    # check stdout
-    subject grep BACALHAU_NODE_NETWORK_ORCHESTRATORS $BACALHAU_DIR/out.log
-    assert_equal 0 $status
-    assert_not_match kerfuffle $stdout
-
-    # check bacalhau.run
-    subject grep BACALHAU_NODE_NETWORK_ORCHESTRATORS $BACALHAU_DIR/bacalhau.run
+    create_node orchestrator
     assert_equal 0 $status
     assert_not_match kerfuffle $stdout
 }
 
 testcase_node_connects_with_preconfigured_token() {
-    subject bacalhau config set node.network.type nats
-    subject bacalhau config set node.network.authsecret kerfuffle
+    subject ${BACALHAU} config set orchestrator.auth.token kerfuffle
     assert_match 0 $status
-    create_node requester
+    create_node orchestrator
 
-    subject bacalhau config set node.network.authsecret kerfuffle
-    subject bacalhau config set node.network.type nats
+    subject ${BACALHAU} config set orchestrator.auth.token kerfuffle
     create_node compute
     # If this returns successfully, the node started and authenticated.
```

Comment:
I don't think this should pass. We are setting `orchestrator.auth.token` which secures the orchestrator node, but not setting `compute.auth.token`. This is making me question if we are actually validating all the tests properly or not

Reply:
not addressed yet!